README.md

```diff
@@ -25,7 +25,31 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+ParseDate is a module with date string parsing methods.
+
+Include the desired functionality in the class of your choice:
+
+```
+require 'parse_date/int_from_string'
+
+class DummyClass
+  include ParseDate::IntFromString
```

Comment:
This is an interesting choice.  Why make this a mixin rather than a plain class or using static methods?  This seems like it leads to mixing concerns in one class. 

I think I'd prefer something like: 
```ruby
ParseDate.year_int_from_date('12/25/00')
#=> 2000
```

Or 
```ruby
parser = ParseDate.new
parser.year_int_from_date('12/25/00')
#=> 2000
```

Reply:
it felt clunky;  that was how I did it in `Stanford::Mods::DateParsing`.   I can change it, sure.  I was thinking about what was optimal and maybe landed in an unusual place.